spec/schema.json

```diff
@@ -25,7 +25,8 @@
             "in": "path",
             "required": true,
             "schema": {
-              "type": "string"
+              "type": "string",
+              "pattern": "^[A-Za-z0-9]{2}[0-9]{6}$"
```

Comment:
This (and the implementation) allows an initial two digits. First two characters can only be letters.

Reply:
See update and comment above.